### ISSUE TITLE
Update the README with working example for ELM 0.17+

### ISSUE THREAD

**Issue by donv** [fbonetti/elm-rails#10]
Hi!

The example in the README gives me the following error when using ELM 0.17.1:

```
-- SYNTAX PROBLEM ------------------------------------- app/assets/elm/Hello.elm

I ran into something unexpected when parsing your code!

1| module Hello where
                ^
I am looking for one of the following things:

    something like `exposing (..)` which replaced `where` in 0.17
    whitespace
```

Since this is my very first attempt at ELM, the first example I encounter should work with the current stable version 😄 .  If a PR is desired, I can do it.

**Comment by Fibrasek** [comment]
You can do the basic `hello-elm` example:

```elm
module Hello exposing (..)

import Html exposing (..)

main =
  div [] [ text "Hello, World!" 
```

:)

**Comment by lsimoneau** [comment]
Resolved in #12
